thirdparty/xmlstream/xmlstream_p.h

```diff
@@ -1140,10 +1140,11 @@ bool XmlStreamReaderPrivate::parse()
             case '\n':
                 ++lineNumber;
                 lastLineStart = characterOffset + readBufferPos;
+                [[gnu::fallthrough]];
```

Comment:
Adjusted. It turns out that something like `// Fall through (for ErrorOnUnexpectedElement)` will not work. Seems that no additional comments must be in the same line.